diff.py

```diff
@@ -71,13 +81,17 @@ def complete_symbol(
                 # skip reading the map file, which would
                 # result in a lot of useless completions
                 return []
-            config: Dict[str, Any] = {}
-            diff_settings.apply(config, parsed_args)  # type: ignore
-            mapfile = config.get("mapfile")
-            if not mapfile:
+            settings: Dict[str, Any] = {}
+            diff_settings.apply(settings, parsed_args)  # type: ignore
+            project = create_project_settings(settings)
```

Comment:
Does this work? I think create_project_settings is defined too far below.

Reply:
yeah I choked